go/vt/tabletserver/tabletconntest/tabletconntest.go

```diff
@@ -828,7 +799,7 @@ var executeBatchQueryResultList = proto.QueryResultList{
 			Fields: []mproto.Field{
 				mproto.Field{
 					Name: "field1",
-					Type: 46,
```

Comment:
It's an invalid type. This function was testing if the type was getting agnostically passed through.
But it's not allowed any more.